quad_control/scripts/utilities/jsonable.py

```diff
@@ -45,10 +45,15 @@ def to_string(cls, inner=dict()):
             arg = args[i+1]
             if arg in cls.inner.keys():
                 val = (inner[arg].__name__, inner[arg].to_string())
+            elif type(defs[i]) is np.ndarray:
+                val = str(list(defs[i]))
```

Comment:
Arrays are encoded as strings. This means that they will look like `parameter_name: "[0.0, 0.0, 0.0]"`.